browser/flagr-ui/src/components/Flag.vue

```diff
@@ -774,16 +775,29 @@ export default {
         this.flag = flag
         this.loaded = true
       }, handleErr.bind(this))
-
-      Axios.get(`${API_URL}/flags/entity_types`).then(response => {
-        let arr = response.data.map(key => {
+      this.fetchEntityTypes()
+    },
+    fetchEntityTypes () {
+      function prepareEntityTypes (entityTypes) {
+        let arr = entityTypes.map(key => {
           let label = key === '' ? '<null>' : key
           return {'label': label, 'value': key}
         })
-        if (response.data.indexOf('') === -1) {
+        if (entityTypes.indexOf('') === -1) {
           arr.unshift({label: '<null>', value: ''})
         }
-        this.entityTypes = arr
+        return arr
+      }
+
+      if (process.env.FLAGR_UI_POSSIBLE_ENTITY_TYPES) {
```

Comment:
FLAGR_UI_POSSIBLE_ENTITY_TYPES

Reply:
It's a bit different, I have to use `process.env.FLAGR_UI_POSSIBLE_ENTITY_TYPES` here, and also it also needs to be registered in `browser/flagr-ui/config`. What the compiler does is to do a string interpolation by using the env variables.